django/utils/text.py

```diff
@@ -434,3 +436,17 @@ def camel_case_to_spaces(value):
     trailing whitespace.
     """
     return re_camel_case.sub(r' \1', value).strip().lower()
+
+
+def _format_lazy(format_string, *args, **kwargs):
+    """
+    Lazy variant of string format, useful when formatting strings with
+    1 or more lazy variables in args and/or kwargs
+    """
+    return force_text(
```

Comment:
Could you add a test that fails if `force_text()` is removed? I'd keep this all on one line.